This file is part of mingw-cross-env.
See doc/index.html for further information.

diff -ru glib-2.23.2.orig/glib-2.0.pc.in glib-2.23.2/glib-2.0.pc.in
--- glib-2.23.2.orig/glib-2.0.pc.in	2009-04-01 01:04:20.000000000 +0200
+++ glib-2.23.2/glib-2.0.pc.in	2010-02-07 23:58:08.000000000 +0100
@@ -3,9 +3,9 @@
 libdir=@libdir@
 includedir=@includedir@
 
-glib_genmarshal=glib-genmarshal
-gobject_query=gobject-query
-glib_mkenums=glib-mkenums
+glib_genmarshal=@bindir@/glib-genmarshal
+gobject_query=@bindir@/gobject-query
+glib_mkenums=@bindir@/glib-mkenums
 
 Name: GLib
 Description: C Utility Library
